templates/learn.md

```diff
@@ -125,6 +125,7 @@ be of interest:
 * [Lean for the Curious Mathematician 2020](https://leanprover-community.github.io/lftcm2020/)
 * [Formal Methods in Mathematics / Lean Together 2020](https://www.andrew.cmu.edu/user/avigad/meetings/fomm2020/)
 * [Lean Together 2019](https://lean-forward.github.io/lean-together/2019/)
+* [Lean for the Curious Mathematician 2022](https://icerm.brown.edu/topical_workshops/tw-22-lean/)
```

Comment:
I think these were in chronological order before

Reply:
it's here #274 I am very sorry for causing trouble for one line.

Reply:
Its fine! PRs are cheap ;)